### PR TITLE
fix issues raised by customers as of 04-17-19

### PR DESCRIPTION
**Pull request by mikeblome**

### CODE REVIEW

File: docs/cpp/config-clang-mac.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ cd helloworld
 code .
 ```
 
-By starting VS Code in a folder, that folder becomes your *workspace*. VS Code stores user settings that are specific to that workspace in `.vscode/settings.json`. In this tutorial, we'll add three additional files to the `.vscode` folder:
+The **code .** command opens VS Code for Mac in the current working directory, which becomes your *workspace*. Our task is to add three files to the workspace that will tell VS Code how to compile and debug our program. VSCode will place these files in a `.vscode` subdirectory that it will create for us:
```
Comment (gregvanl) [inline docs/cpp/config-clang-mac.md]: Italics doesn't look great on code.visualstudio.com so it is better to use bold for term emphasis. **workspace**
Also please use either "Visual Studio Code" (first mention) or "VS Code" (everywhere else in the article). Don't use "VSCode".
There isn't a product called "VS Code for Mac" or "VS Code for Windows", so just use "VS Code".
This comment applies to all four tutorials.

File: docs/cpp/config-clang-mac.md [hunk 2]
```diff
@@ -11,15 +11,15 @@ MetaDescription: Configuring the C++ extension in Visual Studio Code to target C
 
 In this tutorial, you configure Visual Studio Code on macOS to use the Clang/LLVM compiler and debugger. The configuration applies to a single workspace (folder hierarchy), but you can easily copy the configuration files to other workspaces where the same settings are required. After configuring VS Code, you will compile and debug a simple program to get familiar with the VS Code user interface. After completing this tutorial, you will be ready to create and configure your own workspace, and to explore the VS Code documentation for further information about its many features. This tutorial does not teach you about Clang or the C++ language. For those subjects, there are many good resources available on the Web.
 
-If you have any problems, feel free to file an issue for this tutorial in the [VS Code documentation repository](https://github.com/Microsoft/vscode-docs/issues).
+If you have any problems, feel free to file an issue for this tutorial in the [VS Code documentation repository](https://github.com/Microsoft/VS Code-docs/issues).
```
Comment (gregvanl) [inline docs/cpp/config-clang-mac.md]: Beware search and replace. Make sure you don't munge URLs

File: docs/cpp/config-clang-mac.md [hunk 3]
```diff
@@ -11,15 +11,15 @@ MetaDescription: Configuring the C++ extension in Visual Studio Code to target C
...
 
 ## Prerequisites
 
 To successfully complete this tutorial, you must do the following steps:
 
 1. Install [Visual Studio Code](/download) and follow the setup instructions in [Visual Studio Code on macOS](/docs/setup/mac.md).
 
-1. Install the [C++ extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools).
+1. Install the [C++ extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-VS Code.cpptools).
```
Comment (gregvanl) [inline docs/cpp/config-clang-mac.md]: Munged extension identifier

**Comment by mikeblome** [comment]
@gregvanl Getting used to this workflow. in the other repo I'm used to pushing to github even on open PRs to view the commit diffs and then asking for final review with a #sign-off.  In this case, should I have closed the PR after the first change request, then reviewed my commits in private branch before creating a new PR?

**Comment by gregvanl** [comment]
@mikeblome You current workflow is great. I was waiting for the second URL fix before merging.

**Pull request merged**